public/index.html

```diff
@@ -14,7 +14,7 @@
 
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" style="width:300px;"></div>
```

Comment:
Try avoiding inline styling unless it is inevitable. Better - add a class to the css file.
Besides that, this element better stays untouched (it belongs to React). Better - adding the class to the first element inside React's code (probably inside App.tsx)